internal/pkg/model/state.go

```diff
@@ -257,17 +257,21 @@ func (s *State) MustGet(key Key) ObjectState {
 }
 
 func (s *State) CreateFrom(objectManifest Record) (ObjectState, error) {
+	objectState := objectManifest.NewObjectState()
+	return objectState, s.Set(objectState)
+}
+
+func (s *State) Set(objectState ObjectState) error {
```

Comment:
Pridal som metodu `Set` aby sa jednoduchsie pisali testy.
Kedze v produkcnom kode sa pouziva vsade `CreateFrom`, alebo `GetOrCreateFrom`, ... len v testoch je to potom zbytocne neprehladne.